### DISCUSSION TITLE
7 Days to Die how to Latest Experimental ?

### DISCUSSION THREAD

**Discussion by HerrDummkopf** [GameServerManagers/LinuxGSM#3706] [Q&A]
how do I get A20 on the server? with which command

sorry for Google translate^^

**Comment by HerrDummkopf** [comment]
/home/sdtdserver/lgsm/config-lgsm/sdtdserver/sdtdserver.cfg

no working, help pls


##################################
######## Instance Settings ########
##################################
 PLACE INSTANCE SETTINGS HERE
These settings will apply to a specific instance
Carefuly edit this file to add 2 lines on the bottom and make it looks like this:
##################################
######## Instance Settings ########
##################################
PLACE INSTANCE SETTINGS HERE
hese settings will apply to a specific instance

executable="./7DaysToDieServer.x86_64"
experimental_branch="-beta latest_experimental-unstable_build"


________________________________________________
sdtdserver@vmd80387:~$ ./sdtdserver update
[  OK  ] Updating sdtdserver: Checking for update: SteamCMD

No update available
* Local build: 7108526
* Remote build: 7108526
* Branch: public
https://steamdb.info/app/294420/


no update

**Reply by peterkelly70** [reply]
In /home/sevendaystodie/lgsm/config-lgsm/sdtdserver/common.cfg :
## SteamCMD Settings | https://docs.linuxgsm.com/steamcmd : 
# Server appid
appid="294420"
steamcmdforcewindows="no"
# SteamCMD Branch | https://docs.linuxgsm.com/steamcmd/branch
branch="latest_experimental"
betapassword=""
# Master Server | https://docs.linuxgsm.com/steamcmd/steam-master-server
steammaster="true"
## SteamCMD Settings | https://docs.linuxgsm.com/steamcmd
# Server appid
appid="294420"
steamcmdforcewindows="no"
# SteamCMD Branch | https://docs.linuxgsm.com/steamcmd/branch
branch="latest_experimental"
betapassword=""
# Master Server | https://docs.linuxgsm.com/steamcmd/steam-master-server
steammaster="true"

**Comment by HerrDummkopf** [comment]
it worked thank you
